### PR TITLE
fix typo and tweak formatting

### PR DESCRIPTION
**Pull request by lurch**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ about: Create a bug report for the bootloader EEPROM or rpi-eeprom-update script
...
 
 * If you suspect a hardware problem then please read the [Boot Problems](https://www.raspberrypi.org/forums/viewtopic.php?p=437084) post first before contacting the reseller.
-* Support questions or should be posted on the Raspberry Pi [General Discussion](https://www.raspberrypi.org/forums/viewforum.php?f=63)**
+* Support questions or should be posted on the Raspberry Pi [General Discussion](https://www.raspberrypi.org/forums/viewforum.php?f=63) forum
 
 **Mandatory information**
 * Raspberry Pi model
-* Board revision (cat /proc/cpuinfo | grep Revision)
+* Board revision (`cat /proc/cpuinfo | grep Revision`)
 * Operating system version .
```
Comment (pelwell) [inline .github/ISSUE_TEMPLATE/bug_report.md:15]: Spurious `.`

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -34,15 +34,15 @@ vcgencmd bootloader_config
 
 **SD card boot (please complete the following information):**  
  - SD card type
- - Partition information (sudo fdisk -l) if you are able to obtain this from another computer.
+ - Partition information (`sudo fdisk -l`) if you are able to obtain this from another computer.
```
Comment (pelwell) [inline .github/ISSUE_TEMPLATE/bug_report.md:37]: Spurious `.` (there is no verb)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -34,15 +34,15 @@ vcgencmd bootloader_config
...
 
 **USB boot (please complete the following information):**  
-Verify that the the USB device works correctly when hot-plugged under Linux and attache the output of 'lsusb -vvv'
+Verify that the the USB device works correctly when hot-plugged under Linux and attach the output of 'lsusb -vvv'
```
Comment (pelwell) [inline .github/ISSUE_TEMPLATE/bug_report.md:40]: This is an instruction, not a request for information. It is also a senence, and so requires a `.`.

**Comment by lurch** [comment]
Fixed, thanks @pelwell

**Pull request merged**